tests/test_activate.py

```diff
@@ -3357,3 +3357,14 @@ def test_stacking(create_stackable_envs, auto_stack, stack, run, expected):
         )
         == expected
     )
+
+
+def test_activate_and_deactivate_for_uninitialized_env(conda_cli):
+    # Call activate and check that the proper error shows up
+    assert "ERROR: Run 'conda init' before 'conda activate'" in conda_cli("activate")[0]
```

Comment:
We also want to test this with an environment:

```suggestion
    assert "ERROR: Run 'conda init' before 'conda activate'" in conda_cli("activate", "env")[0]
```

Reply:
These messages will only appear in an unconfigured shell. They will not appear in a configured shell with an active environment. I may be missing something though, why do we want to test with an environment?

Reply:
Correct, however thats not what I'm getting at here.

If someone forgot to run `conda init` and tries to do `conda activate env` they will currently get a misleading error message instead of the useful message we want them to get:

```
$ unset conda  # or if using ZSH: `unset -f conda`
$ conda activate env
usage: conda [-h] [-v] [--no-plugins] [-V] COMMAND ...
conda: error: unrecognized arguments: env
```